modules/sr/robot/motor.py

```diff
@@ -75,16 +75,3 @@ def power(self, value):
             value = -SPEED_MAX
 
         self.sr_motor.set_speed(translate(value, self.sr_motor))
-
```

Comment:
Unrelated change? I realise this is dead code, but this doesn't feel like the right PR to be removing it in.